invenio_jsonschemas/config.py

```diff
@@ -36,5 +36,22 @@
 JSONSCHEMAS_REPLACE_REFS = False
 """Whether to resolve $ref before serving a schema."""
 
+JSONSCHEMAS_RESOLVE_SCHEMA = False
+"""Whether to resolve $ref and allOf before serving a schema."""
```

Comment:
If I understand correctly the documentation should be:
```
"""If true, runs the resolver class on the JSON Schema before returning it."""
```

The current documentation should be used to describe the default resolver class.

Reply:
@nharraud Will you make a PR to correct it?

Reply:
I don't have the time right now but I'll create the corresponding issue so that me or somebody else can grab it later.

Reply:
I updated #31 tasks

Reply:
But isn't it just a matter of opening a PR with the line above?

Reply:
IMHO the PR should also document the [`resolve_schema` function](https://github.com/inveniosoftware/invenio-jsonschemas/pull/64/files#diff-e6c80abc800f7eba21ad2f2ba41cf951R33). As said during last meeting "resolving allOf" doesn't mean anything from a JSON-Schema point of view. I can give examples of JSON Schemas where this function doesn't work. We can keep this function as the default one but it should be better documented.

Reply:
I've updated the documentation for both [here](https://github.com/inveniosoftware/invenio-jsonschemas/pull/67). I'd be happy to improve more the documentation of ``resolve_schema`` if you give more infos about the limitations.

Reply:
Thanks @drjova I'll give more explanation in this other PR.